addons/AdvancedUrbanRappelling/functions/fn_AUR_Place_Anchor_Action_Check.sqf

```diff
@@ -1,6 +1,6 @@
 _items = items player;
 _ret = false;
-if (("ToolKit" in _items )and ("AUR_AnchorKit" in _items)) then {
+if ("AUR_AnchorKit" in _items) then {
```

Comment:
einfacher:
```
"AUR_AnchorKit" in (items player) 
```
nur eine Zeile. Bin leider grade etwas aus der klammerung in sqf raus, kann sein das du sogar die hinteren klammern um items player weglassen könntest.

Reply:
good catch